src/lsdb/loaders/dataframe/read_dataframe.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+from hipscat.catalog import CatalogType
+
+from lsdb.catalog.dataset.dataset import Dataset
+from lsdb.loaders.dataframe.dataframe_catalog_loader import DataframeCatalogLoader
+from lsdb.loaders.hipscat.hipscat_loader_factory import CatalogTypeVar
+
+
+def read_dataframe(
+    path: str,
+    catalog_name: str = "",
+    catalog_type: CatalogType | None = None,
+    ra_column: str = "ra",
+    dec_column: str = "dec",
+    threshold: int = 50,
+) -> CatalogTypeVar | Dataset:
```

Comment:
There are a couple more catalog_info parameters, could add a **kwargs that's passed along for those.

Reply:
Done! I've bundled all the _catalog_info_ arguments into kwargs.